content/versions.json

```diff
@@ -1,102 +0,0 @@
-{
-  "targetVersions": [
```

Comment:
Removed need for this file - we get `latest` from `package.json` (hexagon version) and we use the `templates/versions.um` file instead of passing in config